py/strato/common/log/log2text.py

```diff
@@ -122,51 +178,29 @@ def _getNextParsableEntry(inputStream, logFile, colorCode, formatter):
     list the file until the next parsable line
     finish when all lines were listed
     """
+    logTypeConf = formatter._getLogTypeConf(logFile)
     while True:
         try:
             line = inputStream.next()
-            obj = json.loads(line)
-            formatted = formatter.process(obj)
-            return obj, None if formatted is None else _addLogName(formatted, colorCode, logFile)
+            formatted, timestamp = formatter.process(line, logFile, logTypeConf)
+            return None if formatted is None else _addLogName(formatted, colorCode, logFile), timestamp
```

Comment:
i've always returned 2 values (if you'll look at the original code, so did Michael). here i've only switch the places of the values to keep the format (line first, timestamp second).
it uses the timestamp value to chose the first line to print out of the streams.
the None value return is also accounted for, as once the function return None it signals the choosing while loop to ignore the stream
Didn't really change any functionality here though, just refactored the code